PD-17 Clarification on zpool upgrades and Scale (#1809)
truenas/documentation: content/SCALE/SCALETutorials/SystemSettings/UpdateSCALE.md

Select a temporary location to store the update file and click **Choose File**. Select the <file>.iso</file> you want to upgrade to and click **Apply Update**.

## Updating Pools

> After updating, you may find that your storage pools and boot-pool can be updated to enable some supported and requested features are not enabled on the pool.

Reviewer: instead of may, which grants permission, suggest using might as it is possible it could happen and changing to more active voice present tense, maybe something like this?
```suggestion
After updating, you might find that you can update your storage pools and boot-pool to enable some supported and requested features that are not enabled on the pool.
```

> TGo to **System Settings > Shell** and enter `zpool status` to show which pools can be updated.

Reviewer: removed the T, and suggest more present tense active voice
```suggestion
Go to **System Settings > Shell** and enter `zpool status` to show which pools you can update.
```
